README.md

```diff
@@ -193,6 +196,42 @@ EOF
 
 The above CR can be queried by `kubectl get otelinst`.
 
+#### NodeJS
+
+The operator can inject OpenTelemetry [NodeJS auto-instrumentation](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/metapackages/auto-instrumentations-node).
+The injection of the NodeJS instrumentation can be enabled by adding an annotation to the namespace, so that all pods within that namespace will get the instrumentation, or by adding the annotation to individual PodSpec objects, available as part of Deployment, Statefulset, and other resources.
+
+```bash
+instrumentation.opentelemetry.io/inject-nodejs: "true"
+```
+
+The value can be
+* `"false"` - do not inject
+* `"true"` - inject and `Instrumentation` resource from the namespace.
+* `"nodejs-instrumentation"` - name of `Instrumentation` CR instance.
+
+In addition to the annotation, the following `CR` has to be created. The `Instrumentation` resource provides configuration for OpenTelemetry SDK and auto-instrumentation.
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: nodejs-instrumentation
+spec:
+  exporter:
+    endpoint: http://otel-collector:4317
+  propagators:
+    - tracecontext
+    - baggage
+    - b3
+  nodejs:
+    image: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:latest
```

Comment:
@pavolloffay I didn't document this image like you did in Java since I think it's too complicated to create oneself, at least unless someone asks for that.

Reply:
I think, that is fine. I do not expect many people providing custom builds.

I hoped that vendors could use this operator with their agent distros in the future.